PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -22,6 +27,10 @@ if (match) {
 		file_system = new ActiveXObject('Scripting.FileSystemObject'),
 		file = decodeURIComponent(match[1]).replace(/\+/g, ' '),
 		search_path = file.replace(/\//g, '\\');
+		
+	if (settings.project_basepath != '' && settings.project_pathalias != '') {
+		file = file.replace(settings.project_basepath, settings.project_pathalias);

```

Comment:
1. replace `settings.project_basepath` with `settings.projects_basepath`
2. replace `settings.project_pathalias` with `settings.projects_path_alias`
3. replace `file.replace(settings.project_basepath, settings.project_pathalias)` with `file.replace(new RegExp('^' + settings.projects_basepath), settings.projects_path_alias)`

Does that work?